editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx

```diff
@@ -1096,7 +1189,16 @@ export const Card = (props) => (
     <Row style={{ minHeight: 40, gap: 12 }} data-uid="Card-Row-Buttons">
       <Button data-uid="Card-Button-1">Hello</Button>
       <Button data-uid="Card-Button-2">Button</Button>
-      <Button data-uid="Card-Button-3">Button</Button>
+      <Button data-uid="Card-Button-3">${
+        conditional
+          ? `{
+        // @utopia/uid=cond 
```

Comment:
should I see a test case in this file where clicking eventually reaches `.../cond/<expressionUID>`?

Reply:
No, we only select the conditional and not the expression. But with the conditional selected you will be able to edit the content of the active branch.

Reply:
aha! so should I see a test where the path eventually reaches `../cond`?